scripts/rollup/packaging.js

```diff
@@ -235,7 +235,14 @@ function filterOutEntrypoints(name) {
       // Let's remove it.
       files.splice(i, 1);
       i--;
-      unlinkSync(`build/node_modules/${name}/${filename}`);
+      try {
+        unlinkSync(`build/node_modules/${name}/${filename}`);
+      } catch (err) {
+        // If the file doesn't exist we can just move on. Otherwise throw the halt the build
+        if (err.code !== 'ENOENT') {
+          throw err;
+        }
+      }
```

Comment:
this is needed so packaging doesn't crash if the file to be deleted does not exist (which it won't in the stable channel because we don't bother building the external runtime script